xgcm/test/test_metric_ops_multi_axis.py

```diff
@@ -46,15 +44,30 @@ def test_bgrid(self, funcname, boundary, periodic):
             ["dx_t", "dy_t", "dz_t", "area_t", "volume_t"],
             ["xt", "yt", "zt", ["xt", "yt"], ["xt", "yt", "zt"]],
         ):
-            new = func(ds.tracer, axis, **kwargs)
+            if funcname == "cumint":
+                kwargs = dict(boundary=boundary, fill_value=fill_value)
+                new = func(ds.tracer, axis, **kwargs)
+            else:
+                new = func(ds.tracer, axis, boundary, fill_value)
             expected = _expected_result(
-                ds.tracer, ds[metric_name], grid, dim, axis, funcname, **kwargs
+                ds.tracer,
+                ds[metric_name],
+                grid,
+                dim,
+                axis,
+                funcname,
+                boundary,
+                fill_value,
             )
             assert_allclose(new, expected)
 
             # test with tuple input if list is provided
             if isinstance(axis, list):
-                new = func(ds.tracer, tuple(axis), **kwargs)
+                if funcname == "cumint":
+                    kwargs = dict(boundary=boundary, fill_value=fill_value)
+                    new = func(ds.tracer, tuple(axis), **kwargs)
+                else:
+                    new = func(ds.tracer, tuple(axis), boundary, fill_value)
```

Comment:
Similar as above

Reply:
`kwargs` in `cumint` is passed on to `cumsum` which can accept boundary and fill_value
`kwargs` in `average` and `integrate` are passed on to `xarray.mean` and `xarray.sum`
if I just include boundary and fill_value as part of `kwargs` for average and integrate I get the following error statements:
```python
TypeError: nansum() got an unexpected keyword argument 'boundary'
TypeError: mean() got an unexpected keyword argument 'boundary'
```
See lines 1813 and 1865